Utilities/itkSurfaceImageCurvature.hxx

```diff
@@ -86,7 +86,6 @@ SurfaceImageCurvature<TSurface>
   m_UseLabel = false;
   m_kSign = -1.0;
   m_FunctionImage = ITK_NULLPTR;
-  m_Sigma = 1.0;
```

Comment:
The base class defines m_Sigma, and also initializes it, so initialization is not needed in the derived class.